docs/en/Clients-Python.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: global
+title: Python Client
+nickname: Python Client
+group: Clients
+priority: 5
+---
+
+Alluxio has a [Python Client](https://github.com/Alluxio/alluxio-py) for interacting with Alluxio through its
+[REST API](Clients-Rest.html). The Python client exposes an API similar to the [native Java API](Clients-Java-Native.html).
+See the [doc](http://alluxio-py.readthedocs.io) for detailed documentation about all available
+methods. See the [example](https://github.com/Alluxio/alluxio-py/blob/master/example.py) of how to perform basic filesystem
+operations in Alluxio.
+
+# Alluxio Proxy dependency
+
+The Python client talks to Alluxio through the REST API provided by the Alluxio proxy.
+
+The proxy is a standalone server that can be started using
+`${ALLUXIO_HOME}/bin/alluxio-start.sh proxy` and stopped using `${ALLUXIO_HOME}/bin/alluxio-stop.sh
+proxy`. By default, the REST API is available on port 39999.
+
+There are performance implications of using the HTTP proxy. In particular, using the proxy requires
+an extra hop. For optimal performance, it is recommended to run the proxy server and an Alluxio
```

Comment:
Will the "optimal" performance ever be as performant as co-located java client?

Reply:
no, it's as performant as the Go client.